package.json

```diff
@@ -16,7 +16,10 @@
     "flow:vue": "eslint --cache --plugin flowtype-errors --rule 'flowtype-errors/show-errors: error' './frontend/views/**/*.vue'",
     "double": "PORT_SHIFT=1 grunt dev",
     "stylelint": "stylelint 'frontend/**/*.{css,scss,vue}' --fix",
-    "docker": "./scripts/docker.sh"
+    "docker": "./scripts/docker.sh",
+    "dev:dashboard": "grunt --gruntfile Gruntfile.dashboard.js",
+    "build:dashboard": "cross-env NODE_ENV=production grunt build --gruntfile Gruntfile.dashboard.js",
```

Comment:
using `cross-env` here in response to your change request [here](https://github.com/okTurtles/group-income/pull/1476#discussion_r1063847873). But please let me know if you think there is a better way.

Reply:
`cross-env` seems OK. It might be an issue later on that we have to deal with when we switch to Deno though.

Reply:
@taoeffect understood. I have removed `cross-env` and switched it to a command that is only compatible with `Mac` for now, as I still can use both `Windows` and `Mac` for **development** with the current set up.